data_steward/tools/clean_test_project.py

```diff
@@ -0,0 +1,75 @@
+# Python imports
+import logging
+import sys
+
+# Third party imports
+from googleapiclient.errors import HttpError
+
+# Project imports
+import app_identity
+import bq_utils
+
+logging.basicConfig(
+    stream=sys.stdout,
+    level=logging.DEBUG,
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+
+GITHUB_USERNAMES = []
```

Comment:
I assume this is intentionally blank.

Reply:
Yeah but it's better to change it to an argument as you suggested.